codeforamerica/vita-min
# Fyst 1263/require text messages for sms enjoyers

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-1263
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- The form validation explicitly allowed blanks, which I found unusual. I modified that and added a test but I have mild concern that maybe that was intended functionality somehow.

## How to test?
- Navigate through Arizona flow.
- Request SMS notifications and don't input a number
  - Note that you receive an error and cannot proceed.
- Select email notifications
  - Note that you can proceed without entering a phone number
- Click Go Back
- Select SMS notifications
- Enter number
  - Note that you are able to proceed

## Screenshots (for visual changes)
<img width="707" alt="image" src="https://github.com/user-attachments/assets/b7cef2b8-993f-409e-9cf1-d0e13a6ed100">
